public/data/AbhijithGanesh.json

```
@@ -10,7 +10,7 @@
       "icon": "github"
     },
     {
-      "name": "Conenct with me on LinkedIn",
+      "name": "Connenct with me on LinkedIn",
```

There is a typo again.
```suggestion
      "name": "Connect with me on LinkedIn",
```

---

oh thank you